dist/ansible/ovn-playbook.yaml

```diff
@@ -0,0 +1,94 @@
+# Install ovn onto cluster
+# This is done when the cluster is up with no networking
+
+# On a convienent host, clone ovn-kubernetes cd to dist/ansible 
+# git clone https://github.com/openvswitch/ovn-kubernetes ~/ovn-kubernetes
+# cd ~/ovn-kubernetes/dist/ansible
+
+# edit hosts file for the hostname of the master
+
+# ./run-playbook
+
+- hosts: masters
+  become: yes
+  become_user: root
+  tasks:
+  # OVN data is persistent, so delete the database
+  - name: Remove OVS DB Content
+    file:
+      state: absent
+      name: /var/lib/openvswitch/ovnnb_db.db
+  - name: Remove OVS DB Content
+    file:
+      state: absent
+      name: /var/lib/openvswitch/ovnsb_db.db
+
+  # iptables -A OS_FIREWALL_ALLOW -p tcp -m state --state NEW -m tcp --dport 6641 -j ACCEPT
+  - name: iptables allow 6641
+    iptables:
+      chain: OS_FIREWALL_ALLOW
+      ctstate: NEW
+      destination_port: 6641
+      jump: ACCEPT
+      protocol: tcp
+
+  # iptables -A OS_FIREWALL_ALLOW -p tcp -m state --state NEW -m tcp --dport 6642 -j ACCEPT
+  - name: iptables allow 6641
```

Comment:
typo, here should be `iptables allow 6642`

Reply:
good spot. fixed.